packages/report-e2e-tests/src/examples/axe-results-with-issues.snap.html

```diff
@@ -129,6 +129,7 @@
         --nav-link-selected: var(--communication-tint-40);
         --nav-link-expanded: transparent;
       }
+
```

Comment:
**Note:** The changes in the snapshots for the report are only new line changes. This is due to changing the way we manage new line characters in our embedded styles (which itself was necessary for functioning source maps in the details view).